cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -120,7 +134,18 @@ func (c *CStorPoolController) cStorPoolEventHandler(operation common.QueueOperat
 		// try to import/create pool gere as part of resync.
 		if IsPendingStatus(cStorPoolGot) {
 			common.SyncResources.Mux.Lock()
+			if csp != nil {
```

Comment:
Since this code is getting repeated, should we move to a separate function?

Reply:
yes